packages/react/package.json

```diff
@@ -62,17 +63,19 @@
     "@typescript-eslint/parser": "^5.48.2",
     "eslint": "^7.32.0",
     "fs-extra": "^9.0.1",
-    "jest": "^26.6.3",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
`jsdom` is needed to be installed since Jest 26 no longer ships it by default since the test environment is now `node` by default. The test environment needs to be changed to `jsdom` [when building a web app](https://jestjs.io/docs/configuration#testenvironment-string).